packages/cln_plugin/lib/src/json_rpc/response.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'error.dart';
+
+part 'response.g.dart';
+
+@JsonSerializable(explicitToJson: true)
+class Response {
```

Comment:
Fields `result` and `error` are sort of conditional, here as well.

Reply:
same of before